src/tests/test_telemetry.py

```diff
@@ -0,0 +1,76 @@
+from pathlib import Path
+from unittest.mock import ANY, Mock
+import pytest
+import urllib.request
+import duckdb
+from sql.telemetry import telemetry
+from sql import plot
+
+
+@pytest.fixture
+def simple_db_conn():
```

Comment:
We should add a yield here that cleans the file once done.

Reply:
That's the thing I was debating...to reuse the existing db connection for those test cases or each case has its own db session (by yield), I guess the latter it's better approach.